docs/docs/config.md

```diff
@@ -21,6 +21,27 @@ The name of the config file can be changed by setting the following environment
 
 In the example above the new config file name will be: `aether.yaml`
```

Comment:
This isn't a change within your PR, but I noticed this line right above, shouldn't it be `local.yaml` since that is the default aether expects?

Reply:
lol thanks